client/my-sites/sharing/buttons/buttons.jsx

```diff
@@ -107,7 +107,7 @@ class SharingButtons extends Component {
 					initialized={ !! buttons && !! settings }
 					saving={ isSaving } />
 				<ButtonsOptions
-					postTypes={ Object.values( postTypes ) }
+					postTypes={ postTypes }
```

Comment:
Should we just `connect` the `postTypes` prop in `<SharingButtonsOptions />`'s state selector and render `<QueryPostTypes />` there if that's where the data is being used?